kolibri/plugins/management/assets/src/vue/manage-content-page/wizard-export.vue

```diff
@@ -110,6 +135,37 @@
           this.cancelImportExportWizard();
         }
       },
+      bytesForHumans(bytes) {
+        // breaking down byte counts in terms of larger sizes
+        const kilobyte = 1024;
+        const megabyte = Math.pow(kilobyte, 2);
+        const gigabyte = Math.pow(kilobyte, 3);
+
+        function kilobyteCalc(byteCount) {
+          const kilos = Math.floor(byteCount / kilobyte);
+          return `${kilos} KB`;
+        }
+        function megabyteCalc(byteCount) {
+          const megs = Math.floor(byteCount / megabyte);
+          return `${megs} MB`;
+        }
+        function gigabyteCalc(byteCount) {
+          const gigs = Math.floor(byteCount / gigabyte);
+          return `${gigs} GB`;
+        }
+        function chooseSize(byteCount) {
+          if (byteCount > gigabyte) {
+            return gigabyteCalc(byteCount);
+          } else if (byteCount > megabyte) {
+            return megabyteCalc(byteCount);
+          } else if (byteCount > kilobyte) {
+            return kilobyteCalc(byteCount);
+          }
+          return `${bytes} B`;
+        }
+
+        return chooseSize(bytes);
+      },
```

Comment:
I'm seeing some weird behavior here, though it's probably in the python code: https://github.com/learningequality/kolibri/issues/862